### ISSUE TITLE
Font scaling in new gputDrawString() function adds unneeded lines

### ISSUE THREAD

**Issue by d0k3** [Steveice10/ctrcommon#3]
Something seems wrong with the new font scaling in the gputDrawString() function, and it's best seen with the 0xCB symbol (which is a solid block). When the charHeight is > 8 (or even != 8, I tested with 12), additional lines of (transparent) pixels get added to the bottom of the string, making the actual height of the char (plus transparent lines) bigger than charHeight. Or, to put it differently, the character is positioned not at y, but slightly above it.

**Comment by Steveice10** [comment]
Can you provide a code example and possibly visuals describing the issue in detail?

**Comment by d0k3** [comment]
Pretty simple, look at the sample I provided here:
http://wikisend.com/download/884754/ctrfont_test.zip
(sorry, but I didn't kow of any internal file sharing service)

The relevant code is:
int s = 8;
for(int p = 8; (p+s) <  gpuGetViewportWidth(); p += (s++)) {
    gputDrawString(message, p, 2, s, s, 0x00, 0x00, 0x00, 0xFF);
}

In my understanding, all of the letters 'A' should be on the same height (as the y position doesn't change throughout the whole function). Basically, the 'feet' of the letter should be on one horizontal line. However, they are not, making the actual room / position a char takes difficult to control.

I tested this on a N3DS via Ninjhax, if that is relevant information.
